pkg/services/ngalert/provisioning/contactpoints.go

```diff
@@ -491,6 +489,11 @@ groupLoop:
 						// Add the modified receiver to the new group...
 						candidateExistingGroup.GrafanaManagedReceivers = append(candidateExistingGroup.GrafanaManagedReceivers, target)
 						configModified = true
+
+						// if the receiver group turns out to be empty. Remove it.
```

Comment:
```suggestion
						// if the old receiver group turns out to be empty. Remove it.
```
an incredibly tiny nitpick :smile: